Dockerfile

```diff
@@ -1,13 +1,9 @@
 FROM sonarqube:latest
 MAINTAINER Deven Phillips <deven.phillips@redhat.com>
 
-RUN curl -L -o extensions/plugins/sonar-ldap-plugin.jar https://sonarsource.bintray.com/Distribution/sonar-ldap-plugin/sonar-ldap-plugin-2.1.0.507.jar
-RUN curl -L -o extensions/plugins/sonar-github-plugin.jar https://sonarsource.bintray.com/Distribution/sonar-github-plugin/sonar-github-plugin-1.4.1.822.jar
-RUN curl -L -o extensions/plugins/sonar-findbugs-plugin.jar https://github.com/SonarQubeCommunity/sonar-findbugs/releases/download/3.4.4/sonar-findbugs-plugin-3.4.4.jar
-RUN curl -L -o extensions/plugins/sonar-pmd-plugin.jar https://github.com/SonarQubeCommunity/sonar-pmd/releases/download/2.6/sonar-pmd-plugin-2.6.jar
-RUN curl -L -o extensions/plugins/sonar-gitlab-plugin.jar https://github.com/gabrie-allaigre/sonar-gitlab-plugin/releases/download/2.0.1/sonar-gitlab-plugin-2.0.1.jar
-RUN curl -L -o extensions/plugins/sonar-buildbreaker-plugin.jar https://github.com/SonarQubeCommunity/sonar-build-breaker/releases/download/2.2/sonar-build-breaker-plugin-2.2.jar
 RUN cp -a /opt/sonarqube/data /opt/sonarqube/data-init
 RUN cp -a /opt/sonarqube/extensions /opt/sonarqube/extensions-init
 RUN chown 65534:0 /opt/sonarqube && chmod -R gu+rwX /opt/sonarqube
```

Comment:
often folks will put all `RUN` commands into a single `RUN` in order to create less layers. thoughts on that? I don't think its a deal breaker, just curious to know where your head is at on the subject.